packages/backend/src/graphql/schema.graphql

```diff
@@ -181,6 +181,9 @@ type Mutation {
   triggerPaymentPointerEvents(
     input: TriggerPaymentPointerEventsInput!
   ): TriggerPaymentPointerEventsMutationResponse!
+
+  "Update the fee on an asset"
+  updateFee(input: UpdateFeeInput!): UpdateFeeResponse!
```

Comment:
What do you think about something without "update", like `setFee`? Do you think it might be confusing if there's no `feeId` as the input?

Reply:
We should also probably add a `fee` resolver on an asset to be able to look them up